### ISSUE TITLE
Плато Солнечного Колодца (Пророк скверны)

### ISSUE THREAD

**Issue by olegklim1991** [VirginTeam/WotlkCore#816]
**Ссылки и комментарии:**

http://ru.wowhead.com/npc=25038

**Что работает не так?**

**Как должно работать?**

Пророк скверны летает все время на одном месте ,должен летать по кругу как на видео
.Отсутствие на взлетной фазе способности Туман Порчи  http://ru.wowhead.com/spell=45717 ,то есть, она как летит, так и должна разлить этот туман 3 раза ,сначала слево,потом,срава ,потом сново с левой стороны,если попасть в этот туман босс берет тебя в майн контроль(майн  лучи (демонический пар)не нашел в базе, видно ,что не какие големы за игроком не ходят,чисто зеленый луч(убрать големов),что она его использует только тогда когда в небе (у нас ей по барабану где)

**Шаги для воспроизведения ошибки:**

1. Шаг один
2. Шаг два
3. Шаг три

**Текущее обновление ядра:**

Можно посмотреть в игре, введите в чат команду .server info и скопировать строку начиная с Build с номером и датой. Не забывайте указывать, Live Server или PTR Server, где проводилось тестирование.

[//]: # (** Всё, что выделено спецальными символами, это базовая информация о шаблоне баг репорта, её не нужно удалять, она не попадёт в сам репорт и предназначена для удобства оформления репортов. **)
[//]: # (** Этот шаблон предназначен для всех репортов по игровым и серверным проблемам, не редактируйте его и не изменяйте без надобности **)
[//]: # (** Перед созданием репорта прочитайте основную информацию, как оформлять репорты и заголовки. **)
[//]: # (** Приоритеты и ярлыки назначаются только сотрудниками проекта и/или модераторами багтрекера **)
[//]: # (** Ни в коем случае не указывайте название игровой учётной записи или пароль от неё, вы можете оставить имя персонажа для обратной связи с вами **)
[//]: # (** Если вы хотите задать вопрос или уточнить что-то связанное с разработкой, присоединяйтесь к нашему каналу в Discord - https://discord.gg/A7kMHba **)

**Comment by olegklim1991** [comment]
сорян старое видео с оффа удалили не могу найти,но должно работать так как описано

**Comment by olegklim1991** [comment]
https://www.youtube.com/watch?v=6U3DJAhZsQ4 с 3 минуты видео всё видно

**Comment by Senatyan** [comment]
Сейчас слетаю в Санвел с сделаю видео. Сколько не хожу фармить транс, босс всегда летает по кругу. Тут должно быть тоже самое

**Comment by olegklim1991** [comment]
https://www.youtube.com/watch?v=65aZ4917df0 вот еще видео с оффа

**Comment by Senatyan** [comment]
https://www.youtube.com/watch?v=FApO-uG0yak&feature=youtu.be
https://www.youtube.com/watch?v=ih4sqft8E6s&feature=youtu.be
https://www.youtube.com/watch?v=XpL-VicjevI&feature=youtu.be

**Comment by ForesterDev** [comment]
Актуально?

**Comment by Santaria94** [comment]
То, что описано в этом репорте уже исправлено.

**Comment by Plasmashield** [comment]
В последний раз пророк просто сидел на том месте, куда он должен садиться при пуле, а не летал кругами

**Comment by sega693** [comment]
Да, до пула он теперь сидит примерно там, где труп Мандригосы до убийства Бруталла. А не летает. После вайпа садится туда же.
